knotx-adapter/knotx-adapter-common/src/main/java/com/cognifide/knotx/adapter/common/http/HttpClientFacade.java

```diff
@@ -152,9 +150,9 @@ private MultiMap getFilteredHeaders(MultiMap headers, List<Pattern> allowedHeade
         .reduce(Buffer::appendBuffer)
         .doOnNext(this::traceServiceCall)
         .map(buffer -> new ClientResponse()
-            .setBody(buffer)
+            .setBody((io.vertx.core.buffer.Buffer) buffer.getDelegate())
```

Comment:
Yes, this cast is perfectly safe. Rxjava variant of every vertx object wrapps non-rxjava representation in such way - as delegate